### PR TITLE
Elaborate on levels of complexity in the README

### PR DESCRIPTION
**Pull request by omarkohl**

### CODE REVIEW

**Comment by omarkohl** [comment]
@ArranMcC fyi this is copied from (and adapted) from the Google Docs that we were working on in August.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ discussion around AI, ML (machine learning) and AIS in increasing levels of
 complexity to cater to as many readers as possible. We also want to leverage
 analogies, metaphors and drawings to serve as _intuition pumps_.
 
+The three levels of complexity are:
+
+* **Basic**: The target group is people with a high school degree or comparable
+  general knowledge. A useful simplification, since we tend to overcomplicate,
+  is to aim for: _Explain it to me like I am 5_. Even experts can frequently
```
Comment (ArranMcC) [inline README.md:27]: Possibly clarify that you don't mean a literal five-year-old, because there's a big difference between that and a high-school graduate

**Review by ArranMcC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (omarkohl) [reply README.md:27]: Can you make a concrete suggestion? I wrote "A useful simplification, since we tend to overcomplicate ..." which to me already suggests that this is not meant literally.

Comment (ArranMcC) [reply README.md:27]: 'Explain it to me like I am a bright high-school student with limited knowledge of AI.'

Comment (omarkohl) [reply README.md:27]: Are you worried that "Explain it to me like I am 5" could be perceived as offensive or condescending? As far as I know "Explain it like I'm 5" (ELI5) is a common expression that means simplify it and don't use jargon. It is not meant literally. See for example https://xkcd.com/1364/ (which is funny precisely because it interprets the expression literally) or the ELI5 subreddit (https://www.reddit.com/r/explainlikeimfive/) where plenty of concepts are explained that literal five-year old children can't even put in context ("Why does a check bouncing affect the person trying to cash it?").

Replacing it with "Explain it to me like I am a bright high-school student ..." is not a simplification but rather a repetition of the previous sentence "The target group is people with a high school degree ...". I don't think that's valuable. If you feel strongly about this I would remove the entire sentence. In my opinion ELI5 is a useful concept to convey to writers that they should express themselves in simple language without jargon and there is little risk that they will literally try to address children or that someone would be offended by reading this in the README.

Comment (ArranMcC) [reply README.md:27]: I don't think it's offensive or condescending, just inaccurate. If you think that all other potential contributors will know that ELI5 isn't meant to be interpreted literally then you can just leave it as it is. When we worked on the first three articles together, I'm pretty sure the basic level was supposed to literally be aimed at children, hence examples like this: 'For example, three friends who are training to be a scientist, lawyer and athlete respectively will all benefit from time-management skills to juggle their responsibilities, social skills to build useful networks, and a healthy amount of sleep to recover from their busy schedules.' I'm not very familiar with ELI5 and was unaware that it's usually or always supposed to be metaphorical. Regardless it's the README so probably not worth debating further.

**Review by ArranMcC — APPROVED** [review]
